cmd/ddev/cmd/commands.go

```diff
@@ -135,7 +135,7 @@ func addCustomCommands(rootCmd *cobra.Command) error {
 				}
 
 				descSuffix := " (shell " + service + " container command)"
-				if serviceDirOnHost[0:1] == "." {
+				if commandSet == targetGlobalCommandPath {
```

Comment:
The serviceDirOnHost[0:1] was completely wrong. It wasn't just Windows, glad you got this fixed!

Reply:
Also, the way you did it is far more readable, although this whole function is pretty obtuse.